src/Template/Pages/home.ctp

```diff
@@ -35,214 +35,224 @@ $cakeDescription = 'CakePHP: the rapid development PHP framework';
     <title>
         <?= $cakeDescription ?>
     </title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
     <?= $this->Html->meta('icon') ?>
-    <?= $this->Html->css('base.css') ?>
-    <?= $this->Html->css('cake.css') ?>
+    <?= $this->Html->css('home.css') ?>
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:500i|Roboto:300,400,700|Roboto+Mono" rel="stylesheet">
 </head>
-<body class="home">
-    <header>
-        <div class="header-image">
-            <?= $this->Html->image('http://cakephp.org/img/logo-cake.png') ?>
-            <h1>Get the Ovens Ready</h1>
-        </div>
+<body>
+
+    <header class="container">
+        <div class="logo"><?= $this->Html->image('cake-logo.png') ?></div>
+        <h1>Welcome to CakePHP 3.3 Red Velvet. Build fast. Grow solid.</h1>
```

Comment:
Please don'the hard-core the version here.  Ithe will be out of synch in a week.
You can use `Configure::version ()` and only keep the part before the second dot.
I missed it in your previous PR from yesterday